documentation/design-docs/dotnet-tools.md

```diff
@@ -663,6 +664,53 @@ Examples:
     Wrote 12576 bytes to file
     Complete
 
+REPORT
+
+    dotnet-gcdump report -f|--file <input_dump_path>
+    
+    Generate report into stdout from a previously generated gcdump or from a running process.
+    
+    Usage:
+      dotnet-gcdump report [options]
+    
+    Options:
+      -f, --file        The file to read gcdump from.
+      -p, --process-id  The process id to collect the trace.
+      -t, --report-type The type of report to generate. Available options: heapstat (default)
+
+Examples:
+
+    $ dotnet gcdump report -f 20200207_094403_19847.gcdump | head -9
+      5,080,860  GC Heap bytes
+         66,289  GC Heap objects
+
+    Object Bytes  Type
```

Comment:
Its not critical, but including the count of objects would be nice if it is possible. I unresolved the earlier comment about it and added the API that I believe should do the job [MemoryGraph.GetHistogramByType()](https://github.com/dotnet/diagnostics/blob/bc210aa496fd06f94745c82410542e474d953094/src/Tools/dotnet-gcdump/DotNetHeapDump/Graph.cs#L355)

Reply:
I briefly looked for something like that. I'll add that into the report. 👍

Reply:
I am bit confused here, is that `TypeIdx` property corresponds to the index of `memoryGraph.m_types` array?

Reply:
Does this look like a correct approach? just eying the result looks like it's not giving me the correct count of objects.
https://github.com/dotnet/diagnostics/pull/791/files#diff-7fc6cf484dfedf7c72a124738460b1dfR105-R109